File: src/test/java/dev/retrotv/crypto/owe/kdf/scrypt/SCryptTest.java
package dev.retrotv.crypto.owe.kdf.scrypt;

import dev.retrotv.crypto.owe.OWETest;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

public class SCryptTest extends OWETest {

    @Test
    @DisplayName("SCrypt password encode 테스트")
    void passwordEncrypt() {
        passwordEncryptAndMatchesTest(new SCrypt());
    }
}

File: src/test/kotlin/dev/retrotv/crypto/owe/kdf/scrypt/SCryptTest.kt
package dev.retrotv.crypto.owe.kdf.scrypt

import dev.retrotv.crypto.owe.OWETest
import dev.retrotv.crypto.owe.password.scrypt.SCrypt
import org.junit.jupiter.api.DisplayName
import org.junit.jupiter.api.Test

class SCryptTest : OWETest() {
    @Test
    @DisplayName("SCrypt password encode 테스트")
    fun passwordEncrypt() {
        passwordEncryptAndMatchesTest(SCrypt())
    }
}
